test/utils/UintUtils.ts

```diff
@@ -31,11 +31,74 @@ describe('UintUtils', function () {
 
     describe('#toHexString', function () {
       describe('(uint256)', function () {
-        it('todo');
+        it('Should return 0 if input is 0', async () => {
+          const zeroNumber = ethers.BigNumber.from('0');
+          expect(
+            await instance.callStatic['toHexString(uint256)'](zeroNumber),
+          ).to.equal('0x00');
+        });
+
+        it('Should return correct hexString represenation of a number', async () => {
```

Comment:
```suggestion
        it('returns correct hexadecimal string representation of a number', async () => {
```
Spelling, and the word "should" (see above).